### PR TITLE
Compatibility test: create github actions for test (#708)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #708 to release-4.0
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In br repo:
git pr https://github.com/pingcap/br/pull/1012
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/br.git pr/1012:release-4.0-92e7b321d4d8
```

---

### What problem does this PR solve? 
Create s3/gcs compatible actions
### What is changed and how it works?
1. Create github action for compatible test
2. Concurent build cluster latest 3 versions via git tags.
3. Use go-ycsb to generate SQL data.
4. Backup these cluster via SQL backup.
5. Test restore above backup data with newly version one by one.

### Check List 

Tests 

 - Integration test
 - Manual test (add detailed scripts or steps below)

Related changes

 - Need to cherry-pick to the release branch

### Release Note

 - No release note.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -209,3 +209,19 @@ services:
     env_file:
       - ./docker/minio.env
 
+<<<<<<< HEAD
+=======
```
Comment (kennytm) [inline docker-compose.yaml:213]: merge conflict

**Comment by 3pointer** [comment]
/run-integration-tests
```
[2021-04-29T03:57:15.462Z] java.io.IOException: Tried to load head FlowNodes for execution Owner[br_ghpr_unit_and_integration_test/10920:br_ghpr_unit_and_integration_test #10920] but FlowNode was not found in storage for head id:FlowNodeId 1:481
```

File: compatibility/prepare_backup.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+# update tags
+git fetch --tags
+
+TAGS="v5.0.0"
```
Comment (kennytm) [inline compatibility/prepare_backup.sh:21]: should this be changed to v4.0.something

**Review by kennytm — COMMENTED** [review]
rest LGTM

File: compatibility/prepare_backup.sh (hunk 2 above)
Comment (3pointer) [reply compatibility/prepare_backup.sh:21]: This Tag will be overwritten in getLatestTags, so it's ok here?🤔️

Comment (kennytm) [reply compatibility/prepare_backup.sh:21]: ok 🤔

**Pull request merged**